Typescript setup installation issue

I attempted to install this into an existing project:

```
➜  foundryvtt-dnd5eCharacterActions git:(dnd5e-types-revised) ✗ npx @ghost-fvtt/foundry-factory ./
Need to install the following packages:
  @ghost-fvtt/foundry-factory
Ok to proceed? (y) y
Foundry Factory 0.6.5
? Please select whether to create a module or a system: Module
? Target directory /home/andrew/foundryDev/modules/foundryvtt-dnd5eCharacterActions already exists.
 Pick an action: Merge
✔ Created working directory
✔ Existing git repository detected.
? Please pick a preset (links point to the documentation of the preset): ghost's Gulp + Rollup Pres
et
? Check the features needed for your project: TypeScript, Linter & Formatter (ESLint & Prettier), C
SS Pre-processor (Less / SCSS)
? Pick a CSS pre-processor Sass (.scss)
✔ Created programmatic files
✔ Created files from templates
✔ Created additional directories
✖ Failed to install dependencies
Failed to create project at /home/andrew/foundryDev/modules/foundryvtt-dnd5eCharacterActions
Error: Command failed: npm install --loglevel error --save-dev @rollup/plugin-node-resolve chalk fs-extra gulp rollup semver yargs @league-of-foundry-developers/foundry-vtt-types@fvtt-0.7.9 rollup-plugin-typescript2 tslib typescript eslint eslint-config-prettier eslint-plugin-prettier prettier husky lint-staged @typescript-eslint/eslint-plugin @typescript-eslint/parser gulp-dart-sass
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! Found: @league-of-foundry-developers/foundry-vtt-types@0.7.9-6
npm ERR! node_modules/@league-of-foundry-developers/foundry-vtt-types
npm ERR!   dev @league-of-foundry-developers/foundry-vtt-types@"0.7.9-6" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer @league-of-foundry-developers/foundry-vtt-types@"^0.8.8-7" from @league-of-foundry-developers/foundry-vtt-dnd5e-types@1.3.6-2
npm ERR! node_modules/@league-of-foundry-developers/foundry-vtt-dnd5e-types
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /home/andrew/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/andrew/.npm/_logs/2021-07-16T14_18_01_973Z-debug.log
```
Looks like this just needs an update for which types version to install.

--- a/src/presets/ghost-gulp-rollup/ghost-gulp-rollup-preset.ts
+++ b/src/presets/ghost-gulp-rollup/ghost-gulp-rollup-preset.ts
@@ -48,7 +48,7 @@ export class GhostGulpRollupPreset implements Preset {
     let devDependencies = ['@rollup/plugin-node-resolve', 'chalk', 'fs-extra', 'gulp', 'rollup', 'semver', 'yargs'];
     if (this.ghostGulpRollupOptions.useTypeScript) {
       devDependencies = devDependencies.concat([
-        '@league-of-foundry-developers/foundry-vtt-types@fvtt-0.7.9',
+        '@league-of-foundry-developers/foundry-vtt-types@fvtt-0.8.8',
         'rollup-plugin-typescript2',
         'tslib',
         'typescript',

